apps/frontend/src/app/5_pages/ZeroPage/hooks/useLiquityBaseParams.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+import { Decimal } from '@sovryn/utils';
+
+import { getLiquityBaseParams } from '../utils/trove-manager';
+
+export const useLiquityBaseParams = () => {
+  const [liquityBaseParams, setLiquityBaseParams] = useState({
```

Comment:
```suggestion
  const [liquityBaseParams, setLiquityBaseParams] = useState<<ILiquityBaseParams>>({
```
`minBorrowingFeeRate` and `maxBorrowingFeeRate` have an incorrect type (`any`) without this.